main.py

```diff
@@ -1,47 +1,50 @@
-# =============================================================================
-# Importação de bibliotecas
-# =============================================================================
+import os
+import sys
+utils_path = os.path.dirname(os.path.realpath(__file__)) + '\\utils'
+sys.path.insert(0, utils_path)
+
+import print_topic
 import pandas as pd
 pd.set_option('max_colwidth', 400)
 import numpy as np
 import re
 from unidecode import unidecode
```

Comment:
essa lib vai ser usada no futuro?

Reply:
vai sim, xuxu, não precisava ter colocado ela agora mas vamos usar sim, na parte de formatação dos comentários (mas se não for preciso pq o Fabrício não usa, podemos tirar depois tbm)